lib/texter.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Texter
+
+  def self.send_message(message)
+
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    our_number = ENV['OUR_NUMBER']
+    customer_number = ENV['CUSTOMER_NUMBER']
+    @client.account.messages.create({ :from => our_number, :to =>customer_number, :body => message })
```

Comment:
Line is too long. [101/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.